### PR TITLE
Check Ubuntu version in install script

### LINKED ISSUE #110: [installation] Install script should check Ubuntu version 16
The install script should require a minimum Ubuntu version 16.04.

### PR DESCRIPTION
**Pull request by pingwiniasty**
Closes #110

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -4,6 +4,19 @@
 # FUNCTION DEFINTIONS BEGIN
 ###########################
 
+
+check_ubuntu_version() {
+    currentver=`lsb_release -r | awk '{ print $2 }'`
+    requiredver='16.04'
+    if [ "$(printf '%s\n' "$requiredver" "$currentver" | sort -V | head -n1)" = "$requiredver" ];
```
Comment (jarlah) [inline scripts/install.sh]: thats a really nice logic, if its working

1. Get current version from lsb release
2. Sort required version with current version and take the head (given the sort is ascending)
3. check if the head of the sorted version is our required version

This is however only a basic test. If you are running on another debian or ubuntu distro, results might differ ;)

Have you tested it on 14.04 (fail), 16.04 (success) and 18.04 (success)?

Comment (jarlah) [reply scripts/install.sh]: Not directly important to test on 16.04. I can test the function on 14.04 in virtualbox

Comment (jarlah) [reply scripts/install.sh]: I have now tested it. I can verify its working, that it kicks off 14.04. But the way it does that is by running exit. I am not sure if that is user friendly. On my 14.04 it immediately exited the terminal without displaying anything to me. I removed the exit and verified that it reach the else part.

**Comment by pingwiniasty** [comment]
Yup, I tested it on docker with Ubuntu 14.04. I can add function to determine distro

**Comment by pingwiniasty** [comment]
I trying to understand how to stop script execution without terminal termination

**Comment by jarlah** [comment]
At least we have a good enough solution for now

**Pull request merged**